lib/rules/selector-disallowed-list/index.js

```diff
@@ -34,21 +47,44 @@ const rule = (primary) => {
 				return;
 			}
 
-			const { selector, raws } = ruleNode;
+			if (optionsMatches(secondaryOptions, 'ignore', 'inside-block')) {
```

Comment:
[suggest] We can avoid unnecessary repetition of the evaluations by moving the code out of the `walkRules` loop like this:

```js
if (!validOptions) {
	return;
}

const ignoreInsideBlock = optionsMatches(secondaryOptions, 'ignore', 'inside-block');
const iterateThroughList = secondaryOptions && secondaryOptions.iterateThroughList;

root.walkRules((ruleNode) => {
```

Reply:
Ah yes. Thanks!